generators/readme/src/index.ts

```diff
@@ -335,6 +335,16 @@ const parsePropTypesToMarkdown = (componentPath: string) => {
           ];
           break;
         }
+        case 'objectOf': {
```

Comment:
here we support shallow `objectOf`.
Meaning, we will not output nested propType validation details in the case of 

- `objectOf(shape)`
- `objectOf(arrayOf)`
- `objectOf(objectOf)` <- does anyone do this?

Reply:
Seems good for now. We can increase the support of the script as we go.